spinnaker-monitoring-daemon/spinnaker-monitoring/datadog_service.py

```diff
@@ -146,8 +148,16 @@ def read_param(param_name, config_object):
   if host is None and datadog_host is None:
     host = ''
   host = socket.getfqdn(host) if host is not None else datadog_host
-
-  return DatadogMetricsService(api_key=api_key, app_key=app_key, host=host)
+  static_datadog_tags = []
```

Comment:
If this were to be undocumented as a temporary solution (e.g. perhaps this should be configured to the daemon rather than the service so the mechanism applies to any supported service) then I wonder if this should be moved to the constructor so that the use of the environment variable is encapsulated to the service object rather than this factory function. You aernt proposing a means to inject through the factory, so injecting into the object doesnt buy you much flexibility other than unit testing, which could still use the environment variable if you decided to test this.

Speaking of which, you should probably modify the test to show this is adding the right tags. I'd write a test that includes 2 or 3 tags of which 1 is redundant with an explicit tag that is in a test case to show that all the tags are there and not duplicated.